plugins/rapid7_insightvm/help.md

```diff
@@ -14,6 +14,10 @@ This plugin utilizes the [InsightVM API 3](https://help.rapid7.com/insightvm/en-
 
 * Username and password for a user with the necessary permissions
 
+# Supported Product Versions
+
+* Rapid7 InsightVM API v3 2022-03-30T12:00:00Z
```

Comment:
Sorry what you had before was fine (without the time), I'm not sure why I commented what I did before. My fault

Reply:
No problem, I'll change it :)